Week-3/Exercise3.2/Exercise3.2.js

```diff
@@ -0,0 +1,11 @@
+var employee = {
+    firstName: 'John',
+    lastName: 'Doe',
+    getEmpDetails : function(dept, location) {
+        return this.firstName + ' ' + this.lastName + ' works in ' + dept + ' department at ' + location;
+    }
+ }
+
```

Comment:
The example given for bind is correct. But as per guidelines, we have to show all implementation of bind, call & apply. Please add the latter 2 implementation.

Reply:
@chirag-udeshi Also added the call and apply methods implementation

Reply:
Looks good!